README.md

```diff
@@ -454,7 +467,7 @@ class SocialNetworksWidget extends Widget
 }
 ```
 
-### Areas
+### Widget Areas
```

Comment:
Ok, but you should fixed the anchor in the above index [Widget Areas](#widgetareas)